guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
+For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].
+
+[NOTE]
+====
+Click the *Insecure* checkbox when registering a host.
+====
```

Comment:
This is part of global registration, which is also documented elsewhere and doesn't need to be mentioned here.
```suggestion
```